Source/Scene/Points3DTileContent.js

```diff
@@ -340,12 +434,81 @@ define([
         var positions = parsedContent.positions;
         var colors = parsedContent.colors;
         var normals = parsedContent.normals;
+        var batchIds = parsedContent.batchIds;
         var isQuantized = parsedContent.isQuantized;
         var isOctEncoded16P = parsedContent.isOctEncoded16P;
+        var styleableProperties = parsedContent.styleableProperties;
         var isTranslucent = content._isTranslucent;
 
         var hasColors = defined(colors);
         var hasNormals = defined(normals);
+        var hasBatchIds = defined(batchIds);
+
+        var batchTable = content.batchTable;
+        var hasBatchTable = defined(batchTable);
+        var hasStyleableProperties = defined(styleableProperties);
+
+        // TODO : How to expose this? Will this be part of the point cloud styling or a property of the tileset?
+        // Use per-point normals to hide back-facing points.
+        var backFaceCulling = false;
```

Comment:
I changed this to `false` by default since many users will want go inside point clouds, e.g., like the church model.